extensions/ql-vscode/test/vscode-tests/minimal-workspace/variant-analysis/ghec-dr.test.ts

```diff
@@ -0,0 +1,44 @@
+import { ConfigurationTarget } from "vscode";
```

Comment:
I added these tests to `minimal-workspace` because that's where `config.test.ts` is located. I'm not sure if these config tests need a workspace, but as far as I know there's not a huge performance difference between `no-workspace` and `minimal-workspace` so it likely doesn't matter too much.